buildspecs/release-to-maven.yml

```diff
@@ -14,19 +14,31 @@ phases:
   pre_build:
     commands:
     - ROOT=`pwd`
-    - CREDENTIALS=$ROOT/credentials
-    - SETTINGS_XML=$CREDENTIALS/settings.xml
-    - GPG_HOME=$CREDENTIALS/gpghome
+    - CONFIGURATION_ROOT=$ROOT/configuration
+    - SETTINGS_XML_TEMPLATE=$CONFIGURATION_ROOT/settings-template.xml
```

Comment:
Nit: `SETTINGS_XML_TEMPLATE` -> `SETTINGS_TEMPLATE_XML`

Reply:
I don't necessarily agree with that. It's the template for the settings.xml file. The other reads weird to me.